### ISSUE TITLE
sdkVersion raises an error

### ISSUE THREAD

**Issue by thomwiggers** [fidesmo/gradle-javacard#23]
If I put `sdkVersion` in my `build.gradle` like specified in the README, I get this error:

```
No such property: sdkVersion for class: org.gradle.api.internal.initialization.DefaultScriptHandler
```

**Comment by half2me** [comment]
Same here, when running on newer versions of Gradle: 4+

**Comment by dschuermann** [comment]
``sdkVersion`` must be part of the ``javacard`` closure. This is fixed in https://github.com/fidesmo/gradle-javacard/pull/27

**Comment by sergkh** [comment]
Closed by #27
